static/app/views/performance/transactionSummary/filter.tsx

```diff
@@ -79,13 +80,20 @@ function Filter(props: Props) {
   );
 }
 
-const OperationDot = styled('div')<{backgroundColor: string}>`
-  display: block;
-  width: ${space(1)};
-  height: ${space(1)};
-  border-radius: 100%;
-  background-color: ${p => p.backgroundColor};
-`;
+function OperationDot({backgroundColor}: {backgroundColor: string}) {
+  const theme = useTheme();
+  return (
+    <Container
+      width={theme.space.md}
+      height={theme.space.md}
+      alignSelf="center"
+      style={{
+        borderRadius: isChonkTheme(theme) ? theme.radius.full : '100%',
+        backgroundColor,
+      }}
```

Comment:
@JonasBa I think we should at leat be able to apply `borderRadius: 'full'` - the value from our tokens - to `Container`

Regarding `backgroundColor`, not sure; it’s a value that comes from the color palette, so maybe inline style is fine?